include/lsst/afw/typehandling/python.h

```diff
@@ -84,10 +88,121 @@ class StorableHelper : public Base {
     bool equals(Storable const& other) const noexcept override {
         PYBIND11_OVERLOAD_NAME(bool, Base, "__eq__", equals, other);
     }
+
```

Comment:
Preexisting, but up at line 71, I think we should mark that constructor as explicit.  Same for the similar constructor on `PsfTrampoline`.